signal_analog/flow.py

```diff
@@ -181,9 +181,9 @@ def sum(self, by=None, over=None):
         self.call_stack.append(Sum(by=by, over=over))
         return self
 
-    def top(self, by=None, over=None):
+    def top(self, count=None, percentage=None, by=None):
```

Comment:
This isn't backwards compatible.  Should we create a 2.x release branch so we can get all of the api changes in one release?

Reply:
Great suggestion, I think that makes sense.